central/notifier/policycleaner/clean_policy.go

```diff
@@ -0,0 +1,30 @@
+package policycleaner
+
+import "github.com/stackrox/rox/central/detection"
+
+// PolicyCleaner removes notifier from policies.
+type PolicyCleaner struct {
+	buildTimePolicies  detection.PolicySet
+	deployTimePolicies detection.PolicySet
+	runTimePolicies    detection.PolicySet
+}
+
+// DeleteNotifierFromPolicies removes notifier from policies.
+func (s *PolicyCleaner) DeleteNotifierFromPolicies(notifierID string) error {
+	err := s.buildTimePolicies.RemoveNotifier(notifierID)
+	if err != nil {
+		return err
+	}
```

Comment:
Let's not shadow the error here:
```suggestion
	if err := s.buildTimePolicies.RemoveNotifier(notifierID) {
		return err
	}
```